docs/source/tracking.rst

```diff
@@ -292,11 +292,11 @@ Call :py:func:`mlflow.xgboost.autolog` before your training code to enable autom
 
 Autologging captures the following information:
 
-+-----------+------------------------+-----------------------------+---------------+---------------------------------------------------------------------+
-| Framework | Metrics                | Parameters                  | Tags          | Artifacts                                                           |
-+-----------+------------------------+-----------------------------+---------------+---------------------------------------------------------------------+
-| XGBoost   | user-specified metrics | `xgboost.train`_ parameters | --            | `MLflow Model`_ (XGBoost model) on training end; feature importance |
-+-----------+------------------------+-----------------------------+---------------+---------------------------------------------------------------------+
++-----------+------------------------+-----------------------------+---------------+------------------------------------------------------------------------------------+
```

Comment:
is it enough just to add that we now store the input example artifact?

I want to add that the XGBoost model also now contains the signature, but it seems none of the docs for the other flavors mention this